src/oc_erchef/config/vars.config

```diff
@@ -1,7 +1,6 @@
 %% -*- mode: erlang -*-
-{couchdb_server, "localhost"}.
+{couchdb_server, "localhost"}
```

Comment:
Yikes, interseting that this typo snuck in and that it doesn't break anything

Reply:
There should be a period there. That ends the erlang expression. I suspect that file is entirely replaced by the file generated by the chef-server cookbooks, and so this error never blows things up.